packages/vim.rb

```diff
@@ -3,79 +3,106 @@
   })
   binary_sha256 ({
-    aarch64: '7dea736a534604b7356a2056097b13bb67804a9b7281ffae6d8462daa4596096',
-     armv7l: '7dea736a534604b7356a2056097b13bb67804a9b7281ffae6d8462daa4596096',
-       i686: '67422ff9eed379c72a8ce63ee214a5141aee420f187752838efc9c4ade8ffce9',
-     x86_64: '6ebd581a5d978985b1eb8c5fe12fafb8e659e94a311a46a047f0eab2d6e4251b',
   })
 
-  depends_on 'python3'
-  depends_on 'xdg_base'
-
-  def self.preinstall
-    configdir = "#{CREW_PREFIX}/.config"
-    # Save any previous configuration, if it exists.
-    if Dir.exists? "$HOME/.vim"
-      system "cp -r $HOME/.vim #{configdir}"
-      system "mv #{configdir}/.vim #{configdir}/vim"
-      system "rm -rf $HOME/.vim"
-    end
-    if File.exists? "$HOME/.vimrc"
-      system "cp $HOME/.vimrc #{configdir}/vim/vimrc"
-    end
-  end
+  depends_on 'perl' => :build
+  depends_on 'python27' => :build
+  depends_on 'python3' => :build
+  depends_on 'ruby' => :build
 
   def self.build
     system "./configure \
             --prefix=#{CREW_PREFIX} \
-            --enable-gui=no \
+            --localstatedir=#{CREW_PREFIX}/var/lib/vim \
+            --with-features=huge \
+            --with-compiledby='Chromebrew' \
+            --with-tlib=ncurses \
             --without-x \
-            --disable-nls \
+            --disable-gui \
             --enable-multibyte \
-            --with-tlib=ncurses \
-            --enable-perlinterp \
-            --enable-python3interp \
-            --enable-rubyinterp \
-            --with-python3-command=#{CREW_PREFIX}/bin/python3 \
-            --with-ruby-command=#{CREW_PREFIX}/bin/ruby \
-            --with-features=huge \
+            --enable-cscope \
+            --enable-perlinterp=dynamic \
+            --enable-pythoninterp=dynamic \
+            --enable-python3interp=dynamic \
+            --enable-rubyinterp=dynamic \
+            --disable-nls \
             --disable-selinux"
-    system 'make'
+    system "make", "VIMRCLOC=#{CREW_PREFIX}/etc", "VIMRUNTIMEDIR=#{CREW_PREFIX}/share/vim/vim81"
   end
 
   def self.install
-    configdir = "#{CREW_PREFIX}/.config/vim"
-    configdestdir = "#{CREW_DEST_PREFIX}/.config/vim"
     system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
+
     system "compressdoc --gzip -9 #{CREW_DEST_PREFIX}/share/man/man1"
-    FileUtils.mkdir_p "#{configdestdir}"
-    system "cp -r #{CREW_DEST_PREFIX}/share/vim/vim81/* #{configdestdir}"
-    FileUtils.mkdir_p "#{CREW_DEST_HOME}"
-    FileUtils.mkdir_p "#{configdir}" unless Dir.exists? "#{configdir}"
-    system "ln -s #{configdir} #{CREW_DEST_HOME}/.vim"
-    system "ln -s #{configdir}/vimrc #{CREW_DEST_HOME}/.vimrc"
-    system "touch #{configdir}/vimrc #{configdestdir}/vimrc"
+
+    # remove desktop and icon files for the terminal package
+    system "rm", "-r", "#{CREW_DEST_PREFIX}/share/applications"
+    system "rm", "-r", "#{CREW_DEST_PREFIX}/share/icons"
```

Comment:
No need to delete these folders.  They will be automatically removed by crew after installation.

Reply:
@uberhacker These are in `def self.install`. He doesn't want them installed.

Reply:
Exactly, this is reducing the package size, plus they will be provided by the `gvim` package instead.